src/packages/excalidraw/CHANGELOG.md

```diff
@@ -15,6 +15,17 @@ Please add the latest change on the top under the correct section.
 
 ## Excalidraw API
 
+### Features
+
+- Export dialog can be customised with [`UiOptions.canvasActions.export`](https://github.com/excalidraw/excalidraw/blob/master/src/packages/excalidraw/README.md#exportOpts) [#3658](https://github.com/excalidraw/excalidraw/pull/3658).
+
+  Also now [`UIOptions`](https://github.com/excalidraw/excalidraw/blob/master/src/packages/excalidraw/README.md#UIOptions) is memoized to avoid unnecessary rerenders.
+
+#### BREAKING CHANGE
+
+- `UIOptions.canvasActions.saveAsScene` is now renamed to `UiOptions.canvasActions.export.saveFileToDisk`, by default this prop is true hence the **save file to disk** button is rendered inside export dialog.
```

Comment:
```suggestion
- `UIOptions.canvasActions.saveAsScene` was renamed to `UiOptions.canvasActions.export.saveFileToDisk`. Defaults to `true` hence the **save file to disk** button is rendered inside the export dialog.
```

Reply:
shouldn't it be "is" instead of "was" ?

Reply:
it was done in the past. If you wanted to use present tense I think you'd have to use something like `A is now named B`.